### ISSUE TITLE
Presentation mode CSS

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-template-editor#766]
Create CSS profile that is optimized to create screen shots for presentations.

**Comment by willrett** [comment]
I changed the background to be darker to create more contrast.  Let me know if you think we need to do anything else.

**Comment by graybeal** [comment]
I didn't think the contrast issue was for those, based on my understanding from Mark and the context of our meeting last week. And it is not a persistent problem for actual users; it is only a problem for presentations, whether using a screen shot or performing an actual demo.  I think the pre-change colors are perfect for actual users. (Sorry if these points weren't clear, I was talking about them with Marcos and others earlier in the week but didn't realize work had gone forward already. Of course, there may have been some other conversation where I wasn't involved; if so, let's get the people who care together to come to a conclusion.)

The contrast issue is between the text and the background, as in screenshot attached.  The background grey shows up a little dark on projections; and the letters are too small (esp. the content) and too gray (esp. the titles). So the changes I request, only for whenever we make presentations or demos, is a white or almost white background, slightly darker and bold labels, and bold entries that are larger (e.g., same size as labels). 

In discussions a few of us thought the possible solutions, from most user-friendly to least, included:
* user-controllable CSS for at least some of these parameters
* a user-settable "Presenting" flag that would toggle between the above settings and the default ones
* a way developers can change the CSS persistently (across all pages) for a CEDAR session
* a way that a developer can set the CSS in their installation before running CEDAR

The last two don't require development at all, and may be the right answer in the short term. (For the 3rd one, Atti highlights Grease Monkey, if I remember the name correctly, as a way to get persistent CSS changes in the browser. Depending on how well that works, it could be enough for now; but I don't see it as a good solution when we are trying to demonstrate CEDAR in an ad-hoc or dynamic setting.)

<img width="771" alt="screen shot 2018-07-05 at 5 36 07 pm" src="https://user-images.githubusercontent.com/1266896/42354189-11f1a9a0-807a-11e8-9a33-67745fc83b28.png">

**Comment by graybeal** [comment]
Debra  and I discussed the nature of the changes that would be required, and the difficulty of maintaining multiple CSS entries for many parameters.  While the generally recommended approach is to make your site 'more accessible' by making it more usable in unusual situations (e.g., low contrast displays, or projections on small/distant displays), I felt this would 'dumb down' the elegance of the current display. 

So since the first two options above were too difficult, I investigated option 3. Following up on  GreaseMonkey led me eventually to Stylus (https://github.com/openstyles/stylus), which seems to be a pretty elegant CSS styles manager. It lets you persistently change styles used in your browser for a given site, domain, or page. With this I was able to change my view of the CEDAR  workspace and pages.  An example is given at https://github.com/metadatacenter/cedar-docs/wiki/Stylus-Extension-to-Customize-CEDAR-CSS-Displays.

With this capability I think we can close this issue, as we have a way to generate stronger images if needed. We may want to develop a 'standard high-visibility CSS' to make this happen in a consistent way, but we can leave that for another time.
